.github/workflows/test-kctrl-gh.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         ./hack/deploy-test.sh
 
         export KCTRL_E2E_NAMESPACE=kctrl-test
-        export KCTRL_E2E_IMAGE="localhost:5000/test-repo/testimage-$GITHUB_RUN_ID"
+        export KCTRL_E2E_IMAGE="`ifconfig | grep inet | grep -E '\b10\.' | awk '{ print $2}'`:5000/test-repo/testimage-$GITHUB_RUN_ID"
```

Comment:
Here port is 5000 but below it is 8080, not sure what is happening here. I think it should be same. Also, do we really need to export the variable at two places?

Reply:
Yeah that's something missed by me actually in local my 5000 port was occupied so testing it with 8080 and then later forgot to update it. Thanks for bringing this up.

> do we really need to export the variable at two places?
We can start using `cli/hack/test-all-local-registry.sh` script in Github action instead of `cli/hack/test-all.sh` to not to have them in 2 places is it something that you are suggesting?